src/platform/code-editor/package.json

```diff
@@ -40,7 +40,7 @@
     "monaco-editor": "^0.10.1"
   },
   "peerDependencies": {
-    "@angular/common": "^6.0.0-rc.1",
-    "@angular/core": "^6.0.0-rc.1"
+    "@angular/common": "^6.1.0",
+    "@angular/core": "^6.1.0"
```

Comment:
Change these dependencies to `6.0.0`.. we dont need to restrict it to 6.1 and up since we dont use anything in particular of that version.